app/styleguide/wskComponentHandler.js

```diff
@@ -75,6 +75,11 @@ var componentHandler = (function() {
       var registeredClass = findRegisteredClass_(jsClass);
       if (registeredClass) {
         createdComponents_.push(new registeredClass.classConstructor(element));
+        // Call any callbacks the user has registered with this component type.
+        var length = registeredClass.callbacks.length;
+        for (var n = 0; n < length; n++) {
+          registeredClass.callbacks[n](element);
+        }
```

Comment:
Each to their own with regards their personal preferences. I was just curious to your reasoning behind it. Thanks for the discussion. I need to push some more updates, so will swap it over in that shortly, I agree in the expected use cases the gain would be negligible unless there were thousands of callbacks which is unlikely. :-)